### PR TITLE
Change server span 'http-url' value to be absolute path, to be consis…

### PR DESCRIPTION
**Pull request by objectiser**
…tent with value used by the client span

### CODE REVIEW

**Comment by objectiser** [comment]
@kristofa Hi. Small change to make the values used in client and server spans consistent - unless there is a reason that the baseUrl is preferred? If base url is also of interest, possibly it would be stored in another annotation.

Found this when trying out the brave zipkin example, so not sure if the same issue exists with the other frameworks.

**Comment by adriancole** [comment]
Thanks for noticing! Can you do us a favor and do a couple things to prevent this from reoccurring?

* add a unit test for this
* update the javadoc on the interface 

ex. on `com.github.kristofa.brave.http.HttpRequest`

```java
    /**
     * Returns the entire URL, including the scheme, host and query parameters if available.
     *
     * @see zipkin.TraceKeys#HTTP_URL
     */
    URI getUri();
```

**Comment by objectiser** [comment]
@adriancole Hi Adrian, let me know if these changes are sufficient. Thanks.

**Comment by objectiser** [comment]
@adriancole Forgot to ask - when is the next release planned? Any chance of a 3.9.1 in the near future :)

File: brave-resteasy-spring/src/main/java/com/github/kristofa/brave/resteasy/RestEasyHttpServerRequest.java [hunk 1]
```diff
@@ -25,7 +25,8 @@ public String getHttpHeaderValue(String headerName) {
 
     @Override
     public URI getUri() {
-        return req.getUri().getBaseUri();
+    	// Use absolute path, to be consistent with the client span's uri
```
Comment (adriancole) [inline brave-resteasy-spring/src/main/java/com/github/kristofa/brave/resteasy/RestEasyHttpServerRequest.java]: now that the doc is correct, this comment isn't needed

File: brave-resteasy-spring/src/test/java/com/github/kristofa/brave/resteasy/ITBraveResteasy.java [hunk 2]
```diff
@@ -84,6 +88,24 @@ public void test() throws IOException, InterruptedException {
                 serverSpan.getAnnotations().get(0).host.service_name
             );
 
+            // Make sure HTTP URL is present on both spans, and have the same value
```
Comment (adriancole) [inline brave-resteasy-spring/src/test/java/com/github/kristofa/brave/resteasy/ITBraveResteasy.java]: not how I would have tested, but works for me! thanks.

**Comment by adriancole** [comment]
one small change, then I can merge. No problem on a quick release.

**Comment by adriancole** [comment]
there's no planning of releases. we release code when it is needed and vary the version accordingly. I can do a release shortly for you!

**Comment by adriancole** [comment]
3.9.1 on the way!

**Comment by objectiser** [comment]
Thanks

**Pull request merged**